hack/lint-manifests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+readonly OPERATOR_DIR=$1
+readonly CONTROLLER_GEN=$2
+readonly CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+TMP_DIR=$(mktemp -d)
```

Comment:
Sorry I forgot to mention, can you also delete the dir afterwards and just to be sure add it to .gitignore please